docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -36,6 +36,64 @@ The Mongo C++ driver is available in the following package managers.
 - [Vcpkg](https://vcpkg.io/) (search for mongo-cxx-driver)
 - [Conan](https://conan.io/center/recipes/mongo-cxx-driver)
 
+### Vcpkg Install Instructions
+
+If you do not already have Vcpkg installed, install it with the following
+command:
+```
+$ git clone https://github.com/Microsoft/vcpkg.git
+$ cd vcpkg
+$ ./bootstrap-vcpkg.sh
+```
+
+Optionally, to install with Visual Studio integration:
+```
+vcpkg integrate install
+```
+
+Install the driver.
+```
+$ ./vcpkg install mongo-cxx-driver
+```
+
+You can find the header files in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/include/
+```
+
+The library files are in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/lib/
+```
```

Comment:
vcpkg recommends using the `vcpkg.cmake` pre-include file for CMake projects, and prefers the new "manifest mode" when possible. I would note both of those here with external documentation links.